src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class Ball {
+    private Colors color;
+    private int number;
+
+    public Ball(Colors color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public Colors getColor() {
+        return color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setColor(Colors color) {
+        this.color = color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball{"
+                + "color='" + color.toString().toLowerCase() + '\''
```

Comment:
It's better to use name() method for getting String representation of enum constants as described [here](for getting String representation of enum constants)